content/tr/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Cloud Native Sözlüğü"
+status: Completed
+---
+
+**Cloud Native Sözlüğü**
```

Comment:
```suggestion
# Cloud Native Sözlüğü
```

We should better use the same `#` levels for the headers with the English one. https://raw.githubusercontent.com/cncf/glossary/8df5004a00fc1a78ad2914665e691cfee22a6a47/content/en/_index.md


https://gdoc2md.com/ is producing things a bit different.

Reply:
Ah, yes. Thanks. I'll update the rest as well.